src/Tizen.NUI/src/public/XamlBinding/Transition.cs

```diff
@@ -212,5 +231,13 @@ public void AnimateBy(View instance, string behaviorKey)
                 throw new XamlParseException(string.Format("Behaviors don't have key {0}", behaviorKey), new XmlLineInfo());
             }
         }
+
+        /// <since_tizen> 5 </since_tizen>
+        /// This will be public opened in tizen_6.5 after ACR done. Before ACR, need to be hidden as inhouse API.
```

Comment:
In TCSACR-395, I couldn't see any "Add-ACR". so I understand that this is added as Hidden-API, then please **REMOVE** this comment. : `/// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.`
thank you.